### PR TITLE
Crpto-live v1 full development

### PR DESCRIPTION
**Pull request by sylvester-wamaya**
- Set up the project with react, linters, and redux.
- Fetch data from an API.
- Add redux
- Display the coins on the home Page.
- Implement a click event listener to view details of a coin.
- Implement search of coins.
- Add summary.
- Add styling.
- Add testing.

### CODE REVIEW

File: src/__tests__/coinsSlice.test.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable */
```
Comment (Deepakdanger) [inline src/__tests__/coinsSlice.test.jsx]: - [ ] Kindly remove this & solve the linter manually if any linter error.

File: src/__tests__/fetchCoins.test.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/* eslint-disable */
```
Comment (Deepakdanger) [inline src/__tests__/fetchCoins.test.jsx]: - [ ] Kindly remove this & solve the linter manually if any linter error.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @sylvester-wamaya , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
 - [ ] 1. The student recorded a video presenting the features of the project.
2. The maximum length of the video is 5 minutes.
3. The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
4. The video contains a demo of the project, highlighting all the project features.
5. The video contains a highlight about a piece of the code the student finds interesting or is proud of making. 

- [ ] After searching the bitcoin, & selecting a particular one, it is redirecting to the selected bitcoin, But after clicking the back button, it is redirecting to the same search list (that we have searched before). Here after clicking the back button, it should redirect to the home page with all the Bitcoin. Kindly fix it.
- [ ]  _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @sylvester-wamaya , 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**